gef-ui/server/controllers/return-to-maker/index.js

```diff
@@ -39,18 +40,18 @@ export const postReturnToMaker = async (req, res, next) => {
       });
     }
 
-    const comments = [
-      ...(application.comments || []),
-      {
+    if (comment.length > 0) {
+      const commentObj = {
         role: 'checker',
         userName: checker.username,
         createdAt: Date.now(),
         comment,
-      },
-    ];
-
-    application.comments = comments;
-
+      };
+      const comments = application.comments || [];
```

Comment:
was there something wrong with the original implementation here? Needs to be the if there but the rest appears to do the same thing just written in a harder to follow way

Reply:
The previous implementation always added a comment - even if a comment wasn't typed in the form.  
I'll have a look and see if I can simplify.